services/import-service/swagger/swagger.js

```diff
@@ -0,0 +1,31 @@
+// this file was generated by serverless-auto-swagger
+            module.exports = {
```

Comment:
not sure why this swagger folder is here. It's not even implemented in serverless.ts. And the importProductsFile schema itself not match with the real schema. Maybe it would be easier to write some simplified json in pr description, example request, response. etc.